### PR TITLE
K8SPG-53 Use GCS bucket directly

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/53b6eb3a68059cd07041b9bf0995ece914bf363f
image: `perconalab/percona-postgresql-operator:PR-72-53b6eb3a-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-53b6eb3a-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-53b6eb3a-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-53b6eb3a-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-53b6eb3a-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-53b6eb3a-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/97b304bc943fba7a932a711f63a90b1fa9cfe6c8
image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/97b304bc943fba7a932a711f63a90b1fa9cfe6c8
image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-97b304bc-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/e19605da0089d9503aee6ea8e194b5c04614f4a5
image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| affinity | passed |
| demand-backup | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/e19605da0089d9503aee6ea8e194b5c04614f4a5
image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/e19605da0089d9503aee6ea8e194b5c04614f4a5
image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/e19605da0089d9503aee6ea8e194b5c04614f4a5
image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-e19605da-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/d6bf779f510c47a4059dadfe537c281b350f1438
image: `perconalab/percona-postgresql-operator:PR-72-d6bf779f-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-d6bf779f-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-d6bf779f-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-d6bf779f-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-d6bf779f-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-d6bf779f-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/2e22078b0aba681ea96e6aa27e067e813fbdde52
image: `perconalab/percona-postgresql-operator:PR-72-2e22078b-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-2e22078b-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-2e22078b-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-2e22078b-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-2e22078b-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-2e22078b-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/376b11cd43db657b9c72d61eff90df47133b88ba
image: `perconalab/percona-postgresql-operator:PR-72-376b11cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-376b11cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-376b11cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-376b11cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-376b11cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-376b11cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/2bf2016067f568372fd3179108f20c13d1c7fa1d
image: `perconalab/percona-postgresql-operator:PR-72-2bf20160-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-2bf20160-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-2bf20160-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-2bf20160-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-2bf20160-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-2bf20160-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/e2ed04677ff623e716d7d3c37a274dd25bc134cc
image: `perconalab/percona-postgresql-operator:PR-72-e2ed0467-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-e2ed0467-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-e2ed0467-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-e2ed0467-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-e2ed0467-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-e2ed0467-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/e87e8e4645c4e7e823ae754c663540228a9c3b4c
image: `perconalab/percona-postgresql-operator:PR-72-e87e8e46-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-e87e8e46-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-e87e8e46-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-e87e8e46-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-e87e8e46-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-e87e8e46-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| demand-backup | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/aa474fea99dbf799f9b4c28a12c072999a3cc0d1
image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-aa474fea-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/cd5aa7b74847ab897c8e9758c94afaf6c365aca6
image: `perconalab/percona-postgresql-operator:PR-72-cd5aa7b7-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-cd5aa7b7-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-cd5aa7b7-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-cd5aa7b7-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-cd5aa7b7-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-cd5aa7b7-pgo-deployer`

**Review by hors — CHANGES_REQUESTED** [review]

File: e2e-tests/demand-backup/run [hunk 1]
```diff
@@ -46,12 +46,12 @@ function prepare_n_restore {
...
 
-	kubectl_bin get secret/minio-gw-tls -o jsonpath='{.data.tls\.crt}'
-
+	set +x
 	yq w $test_dir/conf/minio-gw-secret.yml 'metadata.name' "${cluster}-backrest-repo-config" \
 		| yq w - 'data[aws-s3-ca.crt]' $(kubectl_bin get secret/minio-gw-tls -o jsonpath='{.data.tls\.crt}') \
+		| yq w - 'data[gcs-key]' $(kubectl_bin get secret/minio-gw-gcs-secret -o jsonpath='{.data.credentials\.json}') \
 		| kubectl_bin apply -f -
-
+	set -x
```
[the remark this answers is not in the document]
Comment (cap1984) [reply e2e-tests/demand-backup/run:55]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/747ee5f94257e3eaf70168f569b9ace01fd11f46
image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-deployer`

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/747ee5f94257e3eaf70168f569b9ace01fd11f46
image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |
| affinity | passed |
| demand-backup | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/72/commits/747ee5f94257e3eaf70168f569b9ace01fd11f46
image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-72-747ee5f9-pgo-deployer`

**Pull request merged**